CrowdFundingApplication/Views/Profile/Settings.cshtml

```diff
@@ -0,0 +1,230 @@
+                                    <label class="form-label" asp-for="Description">О себе</label>
+                                </div>
+                                <div class="mt-3 form-floating">
+                                    <textarea class="form-control" id="sms" name="description" asp-for="Description" style="height: 140px;" placeholder="Расскажите о себе в двух словах!"></textarea>
+                                </div>
+                                <div class="mt-2" style="color: gray;">
+                                    <p style="cursor: pointer">Осталось <label id="result" style="cursor: pointer">100</label> символов</p>
+                                </div>
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
+
+                <div>
+                    <hr />
+                    <div class="row">
+                        <div class="col-lg-8">
+                            <button class="btn btn btn-primary btn-block" style="text-transform: uppercase;" onclick="message();" type="submit">Сохранить изменения</button>
+                            <br />
+                        </div>
+                        <div class="col-lg-4">
+                            <a class="modal-open" style="color: white" data-id="demo-modal">Удалить аккаунт</a>
+                            <br />
+                        </div>
+                    </div>
+                </div>
+
+                <div id="demo-modal" class="modal" role="dialog" tabindex="-1">
+                    <div class="model-inner">
+                        <div class="modal-header">
+                            <h4>Вы действительно хотите удалить аккаунт?</h4>
+                            <button class="modal-close" data-id="demo-modal" aria-label="Close"></button>
+                        </div>
+                        <p class="mt-3">
+                            Вы уверены, что хотите удалить свой аккаунт? В случае, если вы удалите свой аккаунт, все ваши данные
+                            будут немедленно удалены из базы данных!.
+                        </p>
+                        <hr />
+                        <div class="container px-5 ">
+                            <div class="row gx-5">
+                                <div class="col">
+                                    <a class="btn btn-danger btn-block" asp-area="" asp-controller="Profile" asp-action="DeleteProfile">Да</a>
+                                    <br />
+                                </div>
+                                <div class="col">
+                                    <a class="btn btn-success btn-block" asp-area="" asp-controller="Profile" asp-action="Settings">Нет</a>
+                                    <br />
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <script>
```

Comment:
move to js file or js-section https://docs.microsoft.com/en-us/aspnet/core/mvc/views/layout?view=aspnetcore-6.0

Reply:
Has been edited